### DISCUSSION TITLE
spawn a floating term fail(archlinux+alacritty)

### DISCUSSION THREAD

**Discussion by lingllqs** [hyprwm/Hyprland#4235] [General]
bind = SUPER, space, exec, [float;size 1200 800;center] $term 
Yesterday this binding was able to open the floating terminal, today it is not, the configuration has not changed

**Comment by miameows** [comment]
I'm also having this issue. I'd look at the commit history myself, but I'm on my phone since this bug keeps me from using my laptop :)

**Comment by vaxerski** [comment]
git bisect pls

also make sure you are on latest git a similiar bug was fixed yesterday

**Reply by lingllqs** [reply]
It worked fine, but now neovide is having the same problem
